windows/CodePushInvalidUpdateException.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace CodePush.ReactNative
+{
+    class CodePushInvalidUpdateException : Exception
```

Comment:
I think we should remove the "CodePush" prefix, since the namespace makes this clear already.